requirements.txt

```diff
@@ -1,3 +1,5 @@
-opencv-python==4.10.0.84
-viam-sdk==0.29.1
+opencv-python
```

Comment:
Why not pin anymore?

Reply:
yes I can - what is the advantage to doing this?